src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/UsersRepository.cs

```diff
@@ -32,5 +32,10 @@ public async Task CreateUserAsync(ApplicationUser user)
             _context.Users.Add(user);
             await _context.SaveChangesAsync();
         }
+
+        public async Task<ApplicationUser> GetUserById(string userId)
+        {
+            return await _context.Users.FirstOrDefaultAsync(x => x.Id == userId);
+        }
```

Comment:
As a result of some of the higher-level design choices we're starting to see more duplication. For example, this performs the same function as `UserRepository.FindByIdAsync` in the Identity project.

This implementation is absolutely fine – this is really just another prompt that I think we're getting to the point we might want to discuss the impact of some of our design choices so far and potential options.